src/propagation.cpp

```diff
@@ -128,6 +192,14 @@ ot::expected<std::unique_ptr<ot::SpanContext>> SpanContext::deserialize(
           } else if (equals_ignore_case(key, parent_id_header)) {
             parent_id = std::stoull(value);
             parent_id_set = true;
+          } else if (equals_ignore_case(key, sampling_priority_header)) {
+            sampling_priority = asSamplingPriority(std::stoi(value));
+            if (sampling_priority == nullptr) {
+              // The sampling_priority key was present, but the value makes no sense.
+              std::cerr << "Invalid sampling_priority valule in serialized SpanContext"
```

Comment:
`valule`

Reply:
Somehow VS Code keeps disabling spell check, this didn't happen in good old Sublime Text!